force-app/main/default/classes/DMLOperations.cls

```diff
@@ -48,7 +52,10 @@ public with sharing class DMLOperations {
      */
     public static void createAccount(String name, String industry) {
         // Create a new Account and set the fields inline
-
+        Account myAccount = new Account();
+        myAccount.Name = name;
+        myAccount.Industry = industry;
+        insert myAccount;
```

Comment:
Try setting the fields inline versus setting the separately. new Account(Name = ....);